Dockerfile

```diff
@@ -17,6 +17,6 @@ RUN pip install --upgrade pip && \
 
 COPY . /app
 
-RUN DATABASE_URL='' python manage.py collectstatic --noinput
+RUN mv scripts/etl.sh ../bin/metro_etl
```

Comment:
This defines a `metro_etl` task that's executable in a one-off dyno like `heroku run -a la-metro-councilmatic-staging metro_etl`.